### ISSUE TITLE
`exo config`: error: ".../exoscale.toml" exists already             

### ISSUE THREAD

**Issue by tomlankhorst** [exoscale/cli#425]
I can't `exo config`:

```
$ exo version             
exo 1.49.1 7ac7c398 (egoscale 0.84.0)  

$ stat .config/exoscale/exoscale.toml                            
stat: cannot stat '.config/exoscale/exoscale.toml': No such file or directory 

$ exo config                                                     
No Exoscale CLI configuration found                                           
                                                                              
In order to set up your configuration profile, you will need to retrieve      
Exoscale API credentials from your organization's IAM:                        
                                                                              
    https://portal.exoscale.com/iam/api-keys                                  
                                                                              
error: "/home/user/.config/exoscale/exoscale.toml" exists already              
```

**Comment by falzm** [comment]
@tomlankhorst what platform are you running on?

**Comment by falzm** [comment]
I'm not able to reproduce your issue with a test system (fresh Compute instance, latest CLI version installed, no configuration):

```console
ubuntu@test-cli-config:~$ exo version
exo 1.49.1 7ac7c398 (egoscale 0.84.0)

ubuntu@test-cli-config:~$ stat .config/exoscale/exoscale.toml
stat: cannot stat '.config/exoscale/exoscale.toml': No such file or directory

ubuntu@test-cli-config:~$ exo config
No Exoscale CLI configuration found

In order to set up your configuration profile, you will need to retrieve
Exoscale API credentials from your organization's IAM:

    https://portal.exoscale.com/iam/api-keys

[+] API Key [none]: **REDACTED**
[+] Secret Key [none]: **REDACTED**
Retrieving account information... done!

[+] Name [marc@exoscale.com]:
✔ ch-gva-2

ubuntu@test-cli-config:~$ cat .config/exoscale/exoscale.toml
defaultaccount = "**REDACTED**"

[[accounts]]
  account = "**REDACTED**"
  defaultZone = "ch-gva-2"
  endpoint = "https://api.exoscale.com/v1"
  environment = ""
  key = "**REDACTED**"
  name = "**REDACTED**"
  secret = "**REDACTED**"
```

**Comment by tomlankhorst** [comment]
It's Ubuntu 20.04 Server.

I rechecked it in a clean container (ubuntu:focal) it indeed seems to work fine there. 

So I checked what was going on in the ~/.config/exoscale directory, and figured out the permissions were `007`, which later seemed to result from an erroneously set `umask` in a `.bashrc` file. So this had nothing to do with the cli. Sorry for this. 

Thanks a lot for this helpful tool.
